src/app/client/src/app/modules/telemetry/services/telemetry/telemetry.service.ts

```diff
@@ -86,8 +86,9 @@ export class TelemetryService {
    * @param {IStartEventInput} startEventInput
    * @memberof TelemetryService
    */
-  public start(startEventInput: IStartEventInput) {
+  public async start(startEventInput: IStartEventInput) {
     if (this.isInitialized) {
+      startEventInput = await _.cloneDeep(this.addUTM(startEventInput));
```

Comment:
why do we have await here, does _.cloneDeep returns promise or this.addUTM return promise

Reply:
Yes I am waiting for data to reform it properly.